templates/banners/banner.html

```diff
@@ -8,18 +8,31 @@
                 {% endif %}
                 {{ banner.message|render_markdown }}
                 <div>
-                    <button class="btn btn-sm" data-bs-dismiss="alert" aria-label="Close">Close</button>
+                    <button class="btn btn-sm dismiss-banner"
+                            data-banner-id="{{ banner.id }}"
+                            aria-label="Close">Close</button>
                 </div>
             </div>
         {% endfor %}
     </div>
     <script>
         document.addEventListener('DOMContentLoaded', function() {
-            const closeButtons = document.querySelectorAll('[data-bs-dismiss="alert"]');
-            closeButtons.forEach(function(button) {
+            function updateDismissedBanners(bannerId) {
+                const dismissed = JSON.parse(
+                    document.cookie.match(/dismissed_banners=([^;]*)/)?.[1] || '[]'
+                );
```

Comment:
You can use `SiteJS.app.Cookies.get('dismissed_banners')` to get the cookie value. You can also use it to set the value. See the docs: https://github.com/js-cookie/js-cookie